profiles/w3c.js

```diff
@@ -59,6 +59,7 @@ const modules = [
   import("../src/core/anchor-expander.js"),
   import("../src/core/custom-elements/index.js"),
   import("../src/core/web-monetization.js"),
+  import("../src/core/beforeSave.js"),
```

Comment:
Let's follow the file naming conventions we already have (i.e. `before-save.js`)